tests/transactions.test.js

```diff
@@ -0,0 +1,43 @@
+const request = require('supertest');
+const baseUrl = 'localhost:8000';
```

Comment:
Avoid hard coding things that could be different in other people's systems. For instance, my server runs on port 8000 so this test would not run in my system. If you want to hard code it, set it to the default used in the app.js but leave room for configuration with environmental variables

Reply:
Noted. I will make the necessary changes.